api/docs/bt.dox

```diff
@@ -327,6 +327,19 @@ passes an \c instrlist_t of all Level 3 \c instr_t's, for simplicity.
   <tr><td><table width=100% bgcolor="#0000C0" cellspacing=0 cellpadding=1 border=0>
   <tr><td></td></tr></table></td></tr></table></td></tr></table>
 \endhtmlonly
+
+\if level_of_detail
+********************
+\subsection sec_IR_AArch64 Notes about the IR for AArch64
+
+DynamoRIO's IR representation of AArch64 NEON instructions uses an additional
+immediate input operand to denote the width of the vector elements. This is different
+to AArch64 assembly, where the element width is part of the vector register operand.
+For example, floating point vector addition of two vectors with 2 double elements
```

Comment:
Put links to the values of the immediate before the example: otherwise the reader won't know what "0x03" is supposed to mean.  Use # to make a (non-function) link.  E.g., "The immediates take the values #FSZ_HALF, #FSZ_DOUBLE, and #FSZ_SINGLE."

Reply:
Done, thanks!